dotCMS/src/main/java/com/dotcms/content/elasticsearch/util/DotRestHighLevelClientProvider.java

```diff
@@ -146,6 +149,8 @@ private RestClientBuilder getClientBuilder(final BasicCredentialsProvider creden
                     }
                     httpClientBuilder
                             .setDefaultCredentialsProvider(credentialsProvider);
+                    httpClientBuilder.setHostnameVerifier(AllowAllHostnameVerifier.INSTANCE);
```

Comment:
the hostname verifier will always be turned off . Is it what you really want? shouldn't we consider the `ES_TLS_ENABLED` flag?